.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
```

Comment:
This looks like a macOS-specific file (akin to Thumbs.db for Windows users) and probably belongs in the developer's own `.gitignore` instead of an individual project's file.

Reply:
good point - interestingly, this is the `.gitignore` swift package init auto created... fixed.